src/extension.ts

```diff
@@ -233,10 +230,6 @@ class Extension {
       this.terminalProvider?.dispose()
       this.terminalProvider = undefined
 
-      for (const terminal of this.activeTerminals) {
```

Comment:
In general, the manifests in the same project would probably be similar more or less. But we could just show a notification in case this.activeTerminals.size != 0

Reply:
We couldn't reliably track activeTerminals manually since VSCode handles the creation of terminal on the TerminalProvider s unfortunately.

Reply:
We could get the list of terminals though from vscode.window.terminals. Checking which are of the extension's is difficult though to tell.